src/api/java/com/minecolonies/api/configuration/ServerConfiguration.java

```diff
@@ -955,6 +957,9 @@ protected ServerConfiguration(final ForgeConfigSpec.Builder builder)
 
         dynamicTreeHarvestSize = defineInteger(builder, "dynamictreeharvestsize", 5, 1, 5);
 
+        fishingRodDurabilityAdjustT2 = defineInteger(builder, "fishingroddurabilityadjustt2", 6, -249, 250000);
+        fishingRodDurabilityAdjustT1 = defineInteger(builder, "fishingroddurabilityadjustt1", 22, -58, 250000);
```

Comment:
negative values?

Reply:
Comparing them to them ItemTier.WOOD.getMaxUses() and ItemTier.IRON.getMaxUses() for non-fishing tool.  The limits are set such that it won't go less than zero uses as the minimum acceptable use count for a tier.  Such a comparison wouldn't cause harm or a crash, it just wouldn't make sense, since items shouldn't have negative maxUses.

I don't think it's a common use case, but if someone disabled the vanilla fishing rod, had a rod at durability 30, and another at 50, and only wanted the dur-30 rod to be usable by T1 huts, they'd use an adjustT1 value between -10 and -29.

Reply:
Couldn't it potentially go below 0 for rods with less durability than the vanilla ones?

Reply:
The application logic applies the adjustments to the ItemTiers.WOOD and ItemTier.IRON enums, rather than to the instance of the ItemStack for the tool being tested:

> (rodDurability <= (ItemTier.WOOD.getMaxUses() + MinecoloniesAPIProxy.getInstance().getConfig().getServer().fishingRodDurabilityAdjustT1.get())

> (rodDurability <= (ItemTier.IRON.getMaxUses() + MinecoloniesAPIProxy.getInstance().getConfig().getServer().fishingRodDurabilityAdjustT2.get()))

To go below zero, another mod would have to overwrite the ItemTiers, but those are defined in Forge and don't have setters (and I don't think they can be accessed by the datapack system?).

Reply:
okay, sounds good